### DISCUSSION TITLE
battery energy loss calculation

### DISCUSSION THREAD

**Discussion by vdvmichel** [TypQxQ/Sigenergy-Home-Assistant-Integration#33] [Ideas]
2 sensors that keep track of how many kWh the battery has already been charged in its lifespan and also discharged... this way you see the energy loss that we can perhaps express in %. I also have an SMA system with a battery inverter, so not a hybrid system. I see that there is a lot of loss by always converting from DC to AC and vice versa...

**Comment by TypQxQ** [comment]
Did you mean this two?
![image](https://github.com/user-attachments/assets/b38d88d8-7117-4fcc-86f5-a68834e17079)

REgards, Andrei
